tests/test_opencode_wrapper.py

```
@@ -0,0 +1,173 @@
+            env["CMUX_SURFACE_ID"] = "surface:opencode"
+            env["CMUX_SOCKET_PATH"] = str(socket_path)
+        else:
+            env.pop("CMUX_SURFACE_ID", None)
+            env.pop("CMUX_SOCKET_PATH", None)
+        if hooks_disabled:
+            env["CMUX_OPENCODE_HOOKS_DISABLED"] = "1"
+        else:
+            env.pop("CMUX_OPENCODE_HOOKS_DISABLED", None)
+
+        try:
+            result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)